pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/RealtimeSegmentDataManager.java

```diff
@@ -303,7 +314,7 @@ public void run() {
     });
 
     indexingThread.start();
-
+    serverMetrics.addValueToTableGauge(tableName, ServerGauge.SEGMENT_COUNT, 1L);
```

Comment:
If we stop indexing (due to a shutdown), then we need to decrement the count by one, right?